### ISSUE TITLE
Support for Ubuntu 20.04

### ISSUE THREAD

**Issue by halkyon** [TryGhost/Ghost-CLI#1265]
Ubuntu 20.04 is available now, but the stack check when installing Ghost is requiring Ubuntu 16 or 18 specifically.
I've worked around it using `--no-stack` for now.

Doesn't look like any issues with Ubuntu 20.04, but I haven't done an exhaustive check.

**Comment by acburdine** [comment]
@halkyon We've done a bit of looking into Ubuntu 20.04, and the main issue with it at the moment is that, by default, Ubuntu 20.04 installs MySQL 8, instead of MySQL 5.7 (the default for Ubuntu 16 & 18).

Last I checked, there were some compatibility issues between Ghost itself & MySQL 8. Once those compatibility issues are fixed (they may have been fixed already, need to check) and we get any documentation updates done for steps that need to be taken to work on Ubuntu 20, we can look at allowing Ubuntu 20 in the stack check.

**Comment by ErisDS** [comment]
MySQL 8 is in a weird state. There's no official support for it in Node.js yet!

You can read my recent summary of the state of it here: https://github.com/TryGhost/Ghost/issues/11625#issuecomment-644879046

The main things from a Ghost CLI perspective are auth (not sure if it's impacted) and the fact that upgrading from 5 -> 8 will cause mixed collations which is _definitely_ not supported and would never be.

**Comment by dsecareanu** [comment]
Installing ghost on ubuntu 20.04 works using mariadb and having the database credentials setup before hand (though it doesn't help much with the default setup of ubuntu 20.04 and the ghost install command, but at least the 1-click install in DO and other similar options can be update and the install docs could mention this caveat.

**Comment by IvanMoReau** [comment]
> Installing ghost on ubuntu 20.04 works using mariadb and having the database credentials setup before hand (though it doesn't help much with the default setup of ubuntu 20.04 and the ghost install command, but at least the 1-click install in DO and other similar options can be update and the install docs could mention this caveat.

I can actually confirm this.

![Captura de pantalla de 2020-10-02 15-30-26](https://user-images.githubusercontent.com/10843250/94967390-954ec280-04c4-11eb-8188-6156e8f6d38b.png)

**Comment by acburdine** [comment]
Apologies for the delayed response on this - gonna shoot to get basic mysql 8 + ubuntu 20 support out in the next release

**Comment by acburdine** [comment]
Support for MySQL 8 + Ubuntu 20.04 LTS has been merged in and will be released as part of Ghost-CLI 1.15.0

**Comment by acburdine** [comment]
Released Ghost-CLI 1.15.0 a few minutes ago with Ubuntu 20 compatibility - let me know if you see any issues with it 😄

**Comment by david-cako** [comment]
I was not able to get Ghost v4 working on MySQL 8 + 20.04, upgrading from v3@latest.

I tried attaching to an existing database and I got this error during the migrations https://forum.ghost.org/t/upgrading-v4-10-2-to-v4-11-0-error/24903/4 .  Then I tried a fresh Ghost install with a fresh database, using the root user and a password.  It was fine, but when I tried to import my blog contents it showed an error related to nested objects, the output was just [Object object].

On v3 for now, but I would like to upgrade in the future.
